05-Introducing-Rigid-Body-Molecular-Dynamics/03-Thermodynamic-Quantities-of-Rigid-Bodies.ipynb

```diff
@@ -0,0 +1,354 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "650fa61b-d04b-4168-8399-ca0d7a99d616",
+   "metadata": {},
+   "source": [
+    "# Molecular Dynamics Simulations\n",
+    "\n",
+    "## Overview\n",
+    "\n",
+    "### Questions\n",
+    "\n",
+    "* In a rigid body molecular dynamics simulation, how are system quantities measured?  \n",
+    "\n",
+    "### Objectives\n",
+    "\n",
+    "* Define how quanties like the energy are defined in a rigid body molecular dynamics simulation. \n",
+    "* Run a rigid body MD simulation at constant temperature.\n",
+    "* Show that thermodynamic quantities (the total translational kinetic energy in our example) are defined in reference to the mass of the rigid bodies not the constituent particles of the rigid body."
```

Comment:
```suggestion
    "* Show that thermodynamic quantities (the total translational kinetic energy in our example) are defined in reference to rigid bodies not the constituent particles of the rigid body."
```
This is not strictly true. You can access thermodynamic properties of constituent particles it is just not meaningful. The objective is more to show how to get the rigid body thermodynamic properties.